doc/index.rst

```diff
@@ -1,4 +1,69 @@
+===============
+ts_Electrometer
+===============
+A CSC for the Electrometer.
+Released as needed.
 
-.. todo:: add autogen docstrings
+Dependencies
+============
 
-.. todo:: Write documentation
\ No newline at end of file
+.. include:: ../requirements.txt
+
+
+Using ts_Electrometer
+=====================
+This package is compatible with the scons build system and eups package management system.
+
+.. code::
+
+    setup -kr .
+    scons
+    scons install declare
+    package-docs build
+    python bin/RunElectrometer.py 1
+
+Developing ts_Electrometer
+==========================
+Developing the ts_Electrometer package follows the TSSW developer guide.
```

Comment:
Please provide a link to the guide (preferably making "TSSW developer's guide" into the text for the link).